src/actions/albums.js

```diff
@@ -0,0 +1,111 @@
+/* global cozy */
+/**
+  Albums related features
+**/
+
+import {
+  ADD_TO_ALBUM,
+  ADD_TO_ALBUM_SUCCESS,
+  CANCEL_ADD_TO_ALBUM,
+  CREATE_ALBUM_SUCCESS,
+  INDEX_ALBUMS_BY_NAME_SUCCESS
+} from '../constants/actionTypes'
+
+import {
+  ALBUM_DOCTYPE
+} from '../constants/config'
+
+// create album
+export const addToAlbum = (photos = [], album = null) => {
+  return async dispatch => {
+    if (!album) {
+      return dispatch({
+        type: ADD_TO_ALBUM,
+        photos: photos
+      })
+    }
+
```

Comment:
Personal preference I think.

Using promise allow to avoid declaring an `album` var before try/catch.

But I am not close-minded about that, maybe we should agree all together on convention for that kind of stuff. :)